packages/asset-swapper/src/constants.ts

```diff
@@ -83,4 +83,5 @@ export const constants = {
     NULL_ERC20_ASSET_DATA,
     PROTOCOL_FEE_UTILS_POLLING_INTERVAL_IN_MS,
     MARKET_UTILS_AMOUNT_BUFFER_PERCENTAGE,
+    SAMPLER_CONTRACT_GAS_LIMIT: 10e6,
```

Comment:
I'm fine with having this constant in this `constant.ts` file, but I feel like the `GAS_LIMIT` constant makes sense more to be in the `utils/market_operations_utils/constants.ts` file